documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -36,13 +36,13 @@ stringData:
   user: <user> <1>
   password: <password> <2>
   cacert: | <3>
-    <{rhv-short}_ca_certificate>
+    <{rhv_ca_certificate>
```

Comment:
@RichardHoch
1. Please change to "engine_ca_certificate" since the upstream RHV is oVirt and should not be called RHV on upstream doc
2. There is one bracket at the left. please remove it

Reply:
Fixed.